brightray/browser/notification.h

```diff
@@ -33,6 +33,7 @@ struct NotificationOptions {
   bool silent;
   bool has_reply;
   base::string16 reply_placeholder;
+  base::string16 sound_name;
```

Comment:
How about calling this `sound` in accordance with https://developer.mozilla.org/en-US/docs/Web/API/Notification/sound?